admin/src/components/Item/ItemCardHeader/icons.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+// @ts-ignore
+import { Pencil, Trash, Refresh } from '@strapi/icons';
+
+export const pencilIcon = <Pencil />;
+export const refreshIcon = <Refresh />;
+export const trashIcon = <Trash />;
```

Comment:
More question to discuss - are we adding "Icon" at the end of icon component name or we leave it without it? Cause we should set one standard for that

Reply:
I think it is better with the "Icon" suffix. It makes it less possible for a variable name collision to occur.